docs/learn-introduction.md

```diff
@@ -1,9 +1,31 @@
 ---
 id: learn-introduction
-title: Polkadot
-sidebar_label: Polkadot
+title: What is Polkadot?
+sidebar_label: What is Polkadot?
 ---
 
+Polkadot enables scalability by allowing specialized blockchains to communicate with each other in a
+secure, trust-free environment.
+
+Polkadot is built to connect and secure unique blockchains, whether they be public, permission-less
+networks, private consortium chains, or oracles and other Web3 technologies. It enables an internet
```

Comment:
```suggestion
networks, private consortium chains, oracles or other Web3 technologies. It enables an internet
```

Reply:
Just wondering but why should this be changed to 'or' instead of 'and'? =)